src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
```

Comment:
The per day cost of 40 might change in the future. Please consider declaring it as a constant at the top of your function for better maintainability.

e.g. `const price = 40;`

`const totalPrice = days * 40;`